certified-connectors/Signi/apiProperties.json

```diff
@@ -69,7 +69,7 @@
           "description": "The resource you are requesting authorization to use.",
           "constraints": {
             "required": "false",
-            "hidden": true
+            "hidden": "true"

```

Comment:
Please check apiproperties validation errors and change the value of hidden parameter duly.